internal/config/config.go

```diff
@@ -519,11 +519,11 @@ const defaultConfigStr = `
 version: 1
 # What protocol to use when performing git operations. Supported values: ssh, https
 git_protocol: https
-# What editor gh should run when creating issues, pull requests, etc. If blank, will refer to environment.
+# What editor gh should run when creating issues, pull requests, etc. If blank, will refer to environment variable $EDITOR.
```

Comment:
This isn't quite right. If you look at `gh environment` you will see what actually happens:

```
`GH_EDITOR`, `GIT_EDITOR`, `VISUAL`, `EDITOR` (in order of precedence): the editor tool to use
for authoring text.
```